src/MIDebugEngine/Engine.Impl/Breakpoints.cs

```diff
@@ -192,9 +192,16 @@ private static BindResult EvalBindResult(Results bindResult, AD7PendingBreakpoin
             Debug.Assert(bkpt.FindString("type") == "breakpoint");
 
             string number = bkpt.FindString("number");
+            string warning = bkpt.TryFindString("warning");
             string addr = bkpt.TryFindString("addr");
 
-            PendingBreakpoint bp = new PendingBreakpoint(pbreak, number, StringToBreakpointState(addr));
+            PendingBreakpoint bp;
+            if (!string.IsNullOrEmpty(warning))
```

Comment:
Also please update the MI Extensions OneNote page.

---
In reply to: [64833924](https://github.com/Microsoft/MIEngine/pull/346#discussion_r64833924) [](ancestors = 64833924)